lib/routes/drivers/routes.js

```diff
@@ -0,0 +1,130 @@
+import { Sequelize } from 'sequelize';
+import {
+    addDriver,
+    updateDriver,
+    addDriverLocation,
+    acceptBooking,
+    updateBookingAttempts,
+    findBookingAttempt
+} from 'daos/driverDao';
+import { convertDbResponseToRawResponse } from 'utils/transformerUtils';
+import { drivers } from 'utils/apiValidations';
+
+module.exports = [
+    {
+        method: 'POST',
+        path: '/',
+        options: {
+            description: 'register new driver.',
+            notes: 'POST drivers API',
+            tags: ['api', 'drivers'],
+            cors: true,
+            auth: false,
+            validate: drivers.create
+        },
+        handler: async (request, h) => {
+            try {
+                let driver = await addDriver(request.payload);
+                ['createdAt', 'updatedAt'].forEach(key => delete driver[key]);
```

Comment:
This is okay, but what is recommended is do something called specifying attributes. Let's please use that. https://sequelize.org/master/manual/model-querying-basics.html#specifying-attributes-for-select-queries